packages/expo-live-photo/ios/LivePhotoRecords.swift

```diff
@@ -0,0 +1,33 @@
+// Copyright 2024-present 650 Industries. All rights reserved.
+
+import ExpoModulesCore
+import Photos
+
+// swiftlint:disable redundant_optional_initialization - Initialization with nil is necessary
+internal struct LivePhotoAsset: Record {
+  @Field var photoUri: URL? = nil
+  @Field var pairedVideoUri: URL? = nil
+
+  func toLivePhotoStream(targetSize: CGSize, contentFit: ContentFit = .contain) throws -> AsyncThrowingStream<(Bool, PHLivePhoto), Error> {
+    guard
+      let photoUri,
+      let pairedVideoUri
+    else {
```

Comment:
It looks a bit weird, doesn't it? 😅 
```suggestion
    guard let photoUri, let pairedVideoUri else {
```